lib/rubocop/cop/rails/not_null_column.rb

```diff
@@ -44,17 +44,20 @@ def on_send(node)
         private
 
         def check_add_column(node)
-          pairs = add_not_null_column?(node)
-          check_pairs(pairs)
+          add_not_null_column?(node) do |type, pairs|
+            return if type.source.include?('virtual')
```

Comment:
`include?` is a partial match, so I'm worried about false positives. What about below?

```suggestion
            return if type.source == ':virtual'
```

Reply:
It is possible to specify column types as a string (while not as popular as symbol) or a symbol, so I tried to match both. Do you think we can skip it?

Reply:
IMO, It's a corner case, but I think `String#include?` is not suitable because it matches all strings containing `virtual`. This patch will need to add a test case for column types as a string and updating an appropriate logic.